pkgs/node/src/chain.zig

```diff
@@ -168,7 +170,8 @@ test "process and add mock blocks into a node's chain" {
 
     const mock_chain = try stf.genMockChain(allocator, 5, chain_config.genesis);
     const beam_state = mock_chain.genesis_state;
-    var beam_chain = try BeamChain.init(allocator, chain_config, beam_state);
+    const nodeid = 10; // random value
```

Comment:
@g11tech this is one of the test issues I had to fix, please review because I picked something at random

Reply:
yes this is good :+1: